src/os/i18n.zig

```diff
@@ -108,12 +108,15 @@ pub fn canonicalizeLocale(
     // Buffer must be 16 or at least as long as the locale and null term
     if (buf.len < @max(16, locale.len + 1)) return error.NoSpaceLeft;
 
+    // Fix zh locales for macOS
+    const updated = fixZHLocale(locale);
```

Comment:
Pretty sure the Zig convention is to not full-caps acronyms.
>  Acronyms, initialisms, proper nouns, or any other word that has capitalization rules in written English are subject to naming conventions just like any other word. Even acronyms that are only 2 letters long are subject to these conventions. 
```suggestion
    const updated = fixZhLocale(locale);
```